src/main/java/seedu/address/model/person/student/Level.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.model.person.student;
```

Comment:
refactor out of seedu.address.model.person.student to maybe similar to seedu.address.model.tag as class is used by seedu.address.model.tuitionclass